### ISSUE TITLE
SVG Icons have zero width/height

### ISSUE THREAD

**Issue by jchafik** [Eonasdan/tempus-dominus#2527]
- Operating system and version - macOS Monterey 12.3
- Browser and version - Chrome 99.0.4844.83 (arm64)
- Version of the plugin. 6.0.0-beta4
- Demo https://stackblitz.com/edit/tempus-dominus-svg-issue

Using SVG sprites is not currently working as expected. Rendered sprites have zero width and height. Tested with SVG inline in the document as well as external `.svg` documents (same domain and scheme to avoid potential CORS issues).

```
new tempusDominus.TempusDominus(document.getElementById('datetimepicker1'), {
  display: {
    components: {
      clock: false
    },
    icons: {
      type: 'sprites',
      time: '#clock',
      date: '#calendar',
      up: '#arrow-up',
      down: '#arrow-down',
      next: '#chevron-right',
      previous: '#chevron-left',
      today: '#calendar-check',
      clear: '#trash',
      close: '#x',
    },
  },
});
```

**Comment by jchafik** [comment]
I've submitted a PR that resolves this issue.

In the current release, SVG elements are being created in the incorrect namespace, causing them to be interpreted as unknown HTML elements under most circumstances.

I've also added a standard `href` tag to the `use` element to be in line with the current specification. I maintained the now deprecated `xlink:href` for backward compatibility.

See:
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href
https://svgwg.org/svg2-draft/linking.html#XLinkRefAttrs

https://github.com/jchafik/tempus-dominus/commit/12e2a564ef852d2b93635918e5fca7a01acbfdd0
